torch/distributions/transforms.py

```diff
@@ -480,11 +488,18 @@ class ReshapeTransform(Transform):
     Arguments:
         in_shape (torch.Size): The input event shape.
         out_shape (torch.Size): The output event shape.
+        cache_size (int): Size of cache. If zero, no caching is done. If one,
+            the latest single value is cached. Only 0 and 1 are supported.
```

Comment:
Thank you for the doc fix, but please do it in separate PR

Reply:
#144401